### PR TITLE
Adds new integration [meichthys/uptime_kuma]

### PR DESCRIPTION
**Pull request by meichthys**
Link to successful HACS action: [https://github.com/meichthys/uptime_kuma/actions/runs/3231654736](https://github.com/meichthys/uptime_kuma/actions/runs/3231654736)
Link to successful hassfest action (if integration): https://github.com/meichthys/uptime_kuma/actions/runs/3231578053

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -484,6 +484,7 @@
   "mbillow/ha-redpocket",
   "mchwalisz/home-assistant-senec",
   "mdeweerd/zha-toolkit",
+  "meichthys/uptime-kuma",
```
Comment (tjorim) [inline integration:487]:
```suggestion
  "meichthys/uptime_kuma",
```

**Review by tjorim — COMMENTED** [review]
The repo name seems wrong ;)

**Comment by meichthys** [comment]
@tjorim I've released v2.0.0 Is there a way to re-trigger the actions on this repo?

**Comment by meichthys** [comment]
Ah right. I saw that somewhere.

**Comment by meichthys** [comment]
@jayakornk & @m1ckyb would you mind reviewing this to get it past the finish line?

**Pull request closed without merging**